### PR TITLE
Icon btn component

### PR DESCRIPTION
**Pull request by devmarianoguerra**
Created the icons for the Main View header component

### CODE REVIEW

File: dentapp/src/components/UserButton.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+margin: 5px;
+padding: 6px 12px 11px 12px ;
+font-size: 16px;
+`;
+
+function UserButton() {
+  return (
+    <Icon onClick={props.onClick}>
+      <FaUserAlt />
+    </Icon>
```
Comment (diegonava6) [inline dentapp/src/components/UserButton.js]: Add PropTypes

File: dentapp/src/components/shoppingButton.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+background-color: transparent;
+color: #4A4A4A;
+font-size: 20px;
+`;
+
+function ShoppingButton() {
+  return (
+    <Icon onClick={props.onClick}>
+      <MdShoppingCart />
+    </Icon>
```
Comment (diegonava6) [inline dentapp/src/components/shoppingButton.js]: Add PropTypes

**Review by diegonava6 — APPROVED** [review]

**Review by diegonava6 — APPROVED** [review]

File: dentapp/src/components/IconButton.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      userIcon={<FaUserAlt />}
+      shoppingIcon={<MdShoppingCart />}
+    ></Icon>
+  );
+}
+
+export default IconButton;
+
+IconButton.propTypes = {
+  borderRadius: PropTypes.bool.isRequired,
```
Comment (diegonava6) [inline dentapp/src/components/IconButton.js]: call this prop rounded instead of borderRadius

Comment (diegonava6) [reply dentapp/src/components/IconButton.js]: And for what I understand is no required, so it will need a default value, use false

Comment (devmarianoguerra) [reply dentapp/src/components/IconButton.js]: User Icon finished

image

File: dentapp/src/components/IconButton.js [hunk 4]
```diff
@@ -1,39 +1,51 @@
...
 `;
 
-function IconButton() {
-  const { fontSize, borderRadius, color, onClick } = this.props;
+const IconContainer = Styled.div`
+border-radius: ${(props) => props.rounded && "500px"};
+border: ${(props) => props.rounded && "solid 1px " + props.color};
+width: ${(props) => props.rounded && "200px"};
+height: ${(props) => props.rounded && "200px"};
+padding: ${(props) => props.centered && "11px 48px"}
```
Comment (diegonava6) [inline dentapp/src/components/IconButton.js]: I am not sure this centered prop is necessary, the alignment and centering of elements should be handled by the screen using these elements

**Pull request closed without merging**